eisbuk-shared/src/types/firestore.ts

```diff
@@ -42,6 +42,24 @@ import {
  */
 export interface OrganizationData {
   admins: string[];
+  /**
+   * String used as `from` when sending email or `sender` when sending SMS.
+   * If not provided, will fall back to organization name
+   */
+  sender?: string;
+  /**
+   * Template for sent SMS messages
+   */
+  smsConfig?: {
+    /**
+     * Full URL (including host and endpoint) of SMS sending service provider.
+     */
+    url?: string;
+    /**
+     * Auth token used to authenticate with SMS sending service provider
+     */
+    authToken?: string;
```

Comment:
I'm a bit concerned with all the places this auth token will go to. I'd rather prefer it staying home, and never leaving.
Organization attributes are currently sent to the client, but the client doesn't really need them.
I think a better approach could be creating a new `secrets` subcollection in `organizations`. This `secrets` collection should not be readable by any user. Just the allmighty backend code that runs in the firebase cloud functions should be able to read it.
I think this should be part of this PR:
* new rules to deny read permission to anyone to the `secrets` subcolleciton
* adjust code to pull data from said collection

The UI to change these values (not to see/read them) should be part of #102. If possible we could set firestore rules to let admin users write to the `secrets` subcollection, but not read from it. If that's tricky/not possible we could have a cloud function that sets these values when invoked by an admin.